src/components/Product/Product.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import firebase from '@firebase/app';
+function Product() {
+  const [option, setOption] = useState('');
+  const [data, setData] = useState({
+    name: '',
+    lastDate: '',
+  });
+  const addProduct = (event) => {
+    event.preventDefault();
+    console.log('send data...' + data.name, data.lastDate, option);
+    firebase
+      .firestore()
+      .collection('products')
```

Comment:
☝️ This is where we should use the token

In the future, each user will have a `token` stored in their browser - that's what @lizhtml and @AndrewEspejo are working on right now in #25.

![](https://cdn.zappy.app/1c0ebee9cca7a45c1c294f65199d7a4d.png)

Reply:
Is corrected 🥇